lightswind.css

```
@@ -414,17 +412,17 @@ body {
 
 /* Dynamic Navigation Classes */
 .dynamicNav-nav-bg {
```

Stylelint (17.2.0)

[error] Expected class selector ".dynamicNav-nav-bg" to be kebab-case (selector-class-pattern)

(selector-class-pattern)